packages/node_modules/pouchdb-adapter-idb/src/taskQueue.js

```diff
@@ -0,0 +1,44 @@
+// This task queue ensures that IDB open calls are done in their own tick
+// and sequentially - i.e. we wait for the async IDB open to *fully* complete
+// before calling the next one. This works around IE/Edge race conditions in IDB.
```

Comment:
This code was largely copied from `utils.js` but refactored so that it would minify smaller. (e.g. `var running` instead of `taskqueue.running`).